### DISCUSSION TITLE
Change layer display order

### DISCUSSION THREAD

**Discussion by ghost** [Mapsui/Mapsui#2544] [Q&A]
Hello again..

In my .net MAUI application in a ListView I have the list of layers that I have loaded on the map and that I can order at will, with the idea that it changes which one is seen above the ones that are below it in that list.

How can I change the display order on the map so that it looks according to the order it has in the ListView?

Thank you very much in advance.

**Comment by pauldendulk** [comment]
You could use MapControl.Map.Layers.Move(int index, ILayer layer). Internally it removes the layer and inserts it at the new index. 

That is not very easy to do in v4, it needs to be improved.
